src/trading-api/next/v1/bullish-trading-api.yml

```diff
@@ -85,9 +86,9 @@ info:
 
     There are 4 special query parameters used to control the pagination behaviour.
     - `_pageSize` can be one of `5, 25, 50, 100`, default value is `25`
-    - `_nextPage` cursor to the next page
-    - `_previousPage` cursor to the previous page
     - `_metaData` can be false/true, if false the links are NOT part of the response. explicitly set `_metaData=true` to guarantee links are returned.
+    - `_nextPage` cursor to the next page. It is provided in the paginated response when `_metaData=true`.
+    - `_previousPage` cursor to the previous page. It is provided in the paginated response when `_metaData=true`.
```

Comment:
highlights that the value to _nextPage and _previousPage are provided in the response where _metaData=true